docs/utilities/feature_flags.md

```diff
@@ -1,57 +1,366 @@
 ---
-title: Feature flags
-description: Utility
+title: Feature flags description: Utility
 ---
 
-The feature flags utility provides a simple rule engine to define when one or multiple features should be enabled depending on the input.
+The feature flags utility provides a simple rule engine to define when one or multiple features should be enabled
+depending on the input.
 
-!!! tip "For simpler use cases where a feature is simply on or off for all users, use [Parameters](parameters.md) utility instead."
+!!! tip "For simpler use cases where a feature is simply on or off for all users, use [Parameters](parameters.md)
+utility instead."
 
 ## Terminology
 
-Feature flags are used to modify a system behaviour without having to change their code. These flags can be static or dynamic.
+Feature flags are used to modify a system behaviour without having to change their code. These flags can be static or
+dynamic.
 
-**Static feature flags** are commonly used for long-lived behaviours that will rarely change, for example `TRACER_ENABLED=True`. These are better suited for [Parameters utility](parameters.md).
+**Static feature flags** are commonly used for long-lived behaviours that will rarely change, for
+example `TRACER_ENABLED=True`. These are better suited for [Parameters utility](parameters.md).
 
-**Dynamic feature flags** are typically used for experiments where you'd want to enable a feature for a limited set of customers, for example A/B testing and Canary releases. These are better suited for this utility, as you can create multiple conditions on whether a feature flag should be `True` or `False`.
+**Dynamic feature flags** are typically used for experiments where you'd want to enable a feature for a limited set of
```

Comment:
i dont think this is accurate. you can achieve the same thing with static flags. The dynamic part just allows you to change them faster without redeploying your service.

Reply:
Even if we can redploy fast (enough), without rules the flags will have the same value in our code execution, it doesn't make them dynamic. I think we can't achieve dyamic behaivour, say A/B testing, by just redeploying static flags.